docs/src/examples/cavity.md

```diff
@@ -70,13 +70,12 @@ There are two configuration files for this problem, [`cavity_pec.json`]
 (https://github.com/awslabs/palace/blob/main/examples/cavity/cavity_impedance.json).
 
 In both, the [`config["Problem"]["Type"]`](../config/problem.md#config%5B%22Problem%22%5D)
-
 field is set to `"Eigenmode"`, and we use the mesh shown above with a single level of
 uniform mesh refinement (`"UniformLevels": 1`). The material properties for Teflon are
 entered under [`config["Domains"]["Materials"]`]
 (../config/domains.md#domains%5B%22Materials%22%5D). The
-[`config["Domains"]["Postprocessing"]["Dielectric]"`]
-(../config/domains.md#domains["Postprocessing"]["Dielectric"]) object is used to extract the
+[`config["Domains"]["Postprocessing"]["Bulk]"`]
+(../config/domains.md#domains["Postprocessing"]["Energy"]) object is used to extract the
```

Comment:
The link there is from `Bulk` to `Energy`, whereas they were previously both `Dielectric`, is the name mismatch on purpose?